src/Collections/SavedTokens.php

```diff
@@ -44,10 +44,13 @@ public function add($model_data, array $options = [])
      */
     public function create($token_string)
     {
-        $token =  $this->getContainer()->get(
-            SavedToken::class,
-            [(object)['token' => $token_string,], ['collection' => $this,]]
-        );
+        $token_nickname = "token-" . \uniqid();
```

Comment:
I don't seem to have this `uniqid` function. PHP version 7.4.19

Reply:
Possibly a namespacing thing (see `site:list`)
`PHP Fatal error:  Uncaught Error: Call to undefined function Pantheon\Terminus\Models\uniquid() in /Users/tesladethray/Pantheon/terminus/src/Models/OrganizationSiteMembership.php:60`

Reply:
Possibly a namespacing thing (see `site:list`)
`PHP Fatal error:  Uncaught Error: Call to undefined function Pantheon\Terminus\Models\uniquid() in <TERMINUS>/src/Models/OrganizationSiteMembership.php:60`

Reply:
Name spacing/misspelling Fixed.